action.yml

```diff
@@ -20,6 +20,11 @@ inputs:
 runs:
   using: composite
   steps:
+    - shell: bash
+      run: |
+        echo "::notice::NOTE: This action is deprecated and is no longer maintained."
+        echo "::notice::Please, migrate to https://github.com/actions/setup-node"
+
```

Comment:
good idea here!

Reply:
yeah!! it'll print messages in the Actions overview which is perfect. https://github.com/get-bridge/npm-login/actions/runs/1270482220